app/services/idv/analytics_events_enhancer.rb

```diff
@@ -1,112 +1,91 @@
-      idv_in_person_locations_searched
-      idv_in_person_prepare_submitted
-      idv_in_person_prepare_visited
-      idv_in_person_proofing_address_visited
-      idv_in_person_proofing_enrollments_ready_for_status_check_job_completed
-      idv_in_person_proofing_enrollments_ready_for_status_check_job_ingestion_error
-      idv_in_person_proofing_enrollments_ready_for_status_check_job_started
-      idv_in_person_proofing_nontransliterable_characters_submitted
-      idv_in_person_proofing_residential_address_submitted
-      idv_in_person_proofing_state_id_submitted
-      idv_in_person_proofing_state_id_visited
-      idv_in_person_ready_to_verify_sp_link_clicked
-      idv_in_person_ready_to_verify_what_to_bring_link_clicked
-      idv_in_person_send_proofing_notification_attempted
-      idv_in_person_send_proofing_notification_job_completed
-      idv_in_person_send_proofing_notification_job_exception
-      idv_in_person_send_proofing_notification_job_skipped
-      idv_in_person_send_proofing_notification_job_started
-      idv_in_person_switch_back_submitted
-      idv_in_person_switch_back_visited
-      idv_in_person_usps_proofing_enrollment_code_email_received
-      idv_in_person_usps_proofing_results_job_completed
-      idv_in_person_usps_proofing_results_job_deadline_passed_email_exception
-      idv_in_person_usps_proofing_results_job_deadline_passed_email_initiated
-      idv_in_person_usps_proofing_results_job_email_initiated
-      idv_in_person_usps_proofing_results_job_enrollment_incomplete
-      idv_in_person_usps_proofing_results_job_enrollment_updated
-      idv_in_person_usps_proofing_results_job_exception
-      idv_in_person_usps_proofing_results_job_please_call_email_initiated
-      idv_in_person_usps_proofing_results_job_started
-      idv_in_person_usps_proofing_results_job_unexpected_response
-      idv_in_person_usps_proofing_results_job_user_sent_to_fraud_review
-      idv_in_person_usps_request_enroll_exception
-      idv_ipp_deactivated_for_never_visiting_post_office
-      idv_link_sent_capture_doc_polling_complete
-      idv_link_sent_capture_doc_polling_started
-      idv_mail_only_warning_visited
-      idv_native_camera_forced
-      idv_not_verified_visited
-      idv_phone_use_different
-      idv_request_letter_visited
-      idv_sdk_selfie_image_capture_closed_without_photo
-      idv_sdk_selfie_image_capture_failed
-      idv_sdk_selfie_image_capture_opened
-      idv_selfie_image_added
-      idv_session_error_visited
-      idv_threatmetrix_response_body
-      idv_usps_auth_token_refresh_job_completed
-      idv_usps_auth_token_refresh_job_network_error
-      idv_usps_auth_token_refresh_job_started
-      idv_verify_by_mail_enter_code_submitted
-      idv_verify_by_mail_enter_code_visited
-      idv_verify_in_person_troubleshooting_option_clicked
-      idv_warning_action_triggered
-      idv_warning_shown
-    ].to_set.freeze
+    EXCLUDED_FRONTEND_EVENT_METHODS = [
```

Comment:
Was there a motivation in switching from symbol array literal `%i[` to plain-ol' array of symbols? I quite like how the symbol array removes a lot of the noise with `:` and `,`.

(I also can't help but notice that [Rubocop has a cop we can use](https://docs.rubocop.org/rubocop/cops_style.html#stylesymbolarray) if we're feeling opinionated one way or the other about its use)

Reply:
Not really. I find that my particularly editor doesn't really know how to fold the region of %-style Array literals, but I'm fine with changing it back.

Reply:
Reverted to using %-style array literal syntax